include/seqan3/core/simd/detail/simd_algorithm_avx2.hpp

```diff
@@ -81,9 +81,59 @@ constexpr simd_t load_avx2(void const * mem_addr)
     return reinterpret_cast<simd_t>(_mm256_loadu_si256(reinterpret_cast<__m256i const *>(mem_addr)));
 }
 
-// TODO: not implemented and used yet, if you implement it don't forget to add it to seqan3::simd::transpose
```

Comment:
I guess this was done or is not applicable anymore?

Reply:
it is implemented now.